shell/platform/tizen/tizen_renderer_evas_gl.cc

```diff
@@ -0,0 +1,702 @@
+        "EVAS_GL_GLES_2_X,");
+    gl_context_ =
+        evas_gl_context_version_create(evas_gl_, NULL, EVAS_GL_GLES_2_X);
+    gl_resource_context_ =
+        evas_gl_context_version_create(evas_gl_, gl_context_, EVAS_GL_GLES_2_X);
+  }
+  if (gl_context_ == nullptr) {
+    FT_LOGE("Failed to create evas gl context!");
+    FT_RELEASE_ASSERT_NOT_REACHED();
+  }
+
+  EVAS_GL_GLOBAL_GLES3_USE(g_evas_gl, gl_context_);
+  gl_surface_ = evas_gl_surface_create(evas_gl_, gl_config_, w, h);
+
+  gl_resource_surface_ =
+      evas_gl_pbuffer_surface_create(evas_gl_, gl_config_, w, h, NULL);
+
+  Evas_Native_Surface ns;
+  evas_gl_native_surface_get(evas_gl_, gl_surface_, &ns);
+  evas_object_image_native_surface_set((Evas_Object*)GetImageHandle(), &ns);
+
+  return true;
+}
+
+void* TizenRendererEvasGL::SetupEvasWindow(int32_t x, int32_t y, int32_t w,
+                                           int32_t h) {
+  if (w == 0 || h == 0) {
+    FT_LOGE("Failed to create because of the wrong size");
+    return nullptr;
+  }
+  elm_config_accel_preference_set("hw:opengl");
+
+  evas_window_ = elm_win_add(NULL, NULL, ELM_WIN_BASIC);
+  elm_win_alpha_set(evas_window_, EINA_FALSE);
+  evas_object_move(evas_window_, x, y);
+  evas_object_resize(evas_window_, w, h);
+  evas_object_raise(evas_window_);
+
+  Evas_Object* bg = elm_bg_add(evas_window_);
+  evas_object_color_set(bg, 0x00, 0x00, 0x00, 0x00);
+
+  evas_object_size_hint_weight_set(bg, EVAS_HINT_EXPAND, EVAS_HINT_EXPAND);
+  elm_win_resize_object_add(evas_window_, bg);
+
+  graphics_adapter_ =
+      evas_object_image_filled_add(evas_object_evas_get(evas_window_));
+  evas_object_resize(graphics_adapter_, w, h);
+  evas_object_move(graphics_adapter_, x, y);
+  evas_object_image_size_set(graphics_adapter_, w, h);
+  evas_object_image_alpha_set(graphics_adapter_, EINA_TRUE);
+  elm_win_resize_object_add(evas_window_, graphics_adapter_);
+
+  int rotations[4] = {0, 90, 180, 270};
+  elm_win_wm_rotation_available_rotations_set(evas_window_,
+                                              (const int*)(&rotations), 4);
+  evas_object_smart_callback_add(evas_window_, "rotation,changed",
+                                 RotationEventCb, this);
+  return (void*)evas_window_;
+}
+
+void TizenRendererEvasGL::DestroyEvasGL() {
+  evas_gl_surface_destroy(evas_gl_, gl_surface_);
+  evas_gl_surface_destroy(evas_gl_, gl_resource_surface_);
+
+  evas_gl_context_destroy(evas_gl_, gl_context_);
+  evas_gl_context_destroy(evas_gl_, gl_resource_context_);
+
+  evas_gl_config_free(gl_config_);
+  evas_gl_free(evas_gl_);
+}
+
+void TizenRendererEvasGL::DestroyEvasWindow() {
+  evas_object_del(evas_window_);
+  evas_object_del(graphics_adapter_);
+}
+
+void TizenRendererEvasGL::RotationEventCb(void* data, Evas_Object* obj,
+                                          void* event_info) {
+  auto* self = reinterpret_cast<TizenRendererEvasGL*>(data);
+  // TODO : Use current window rotation degree
+  FT_UNIMPLEMENTED();
```

Comment:
Note : Implementation was not successful when implementing PoC. I've marked it, but we have to implement it someday.

Reply:
This is registered in the issue #71